pages/_type/_id.vue

```diff
@@ -1193,6 +1230,17 @@ export default {
         this.$nuxt.$loading.finish()
       }
     },
+    async getLicenseData() {
+      try {
+        const text = await this.$axios.get(
+          `tag/license/${this.project.license.id}`
+        )
+        this.licenseText = text.data.body
+        this.$refs.modal_license.show()
+      } catch {
+        this.licenseText = 'License text could not be retrieved.'
```

Comment:
Shouldn't the modal show even though the request fails? Just so they can see this message